oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -611,9 +621,13 @@ private IdToken getIdTokenOauthEndpoint(String targetAudience) throws IOExceptio
         MetricsUtils.getGoogleCredentialsMetricsHeader(
             RequestType.ID_TOKEN_REQUEST, getMetricsCredentialType()));
 
+    LoggingUtils.logRequest(
+        request, LOGGER_PROVIDER, "Sending request to get ID token via Oauth endpoint");
```

Comment:
```suggestion
        request, LOGGER_PROVIDER, "Sending request to get ID token");
```

I think it would be better to not include the internal implementation details. I think it would be better to keep it as `... retrieving ID Token` or similar.

Reply:
hmm, I feel like this could be helpful information for debugging. 
Also, fyi - we currently publish client debug logging with disclaimer. And we will be able to make changes to these log messages without breaking change.

Reply:
Users don't get to configure any settings to choose whether to hit the Oauth or IAM endpoint. It's an implementation detail and we decide based on the UD value of the Credentials. As long as users see that it's trying to retrieve the ID token, I think that's should suffice for the log.

The `request.url` value should be populated as part of the structure log IIUC and that could be helpful to determine exactly which endpoint is sent.